src/invariant/LedgerEntryIsValid.cpp

```diff
@@ -368,4 +370,41 @@ LedgerEntryIsValid::checkIsValid(LedgerEntry const& le,
 
     return {};
 }
+
+std::string
+LedgerEntryIsValid::checkIsValid(LiquidityPoolEntry const& lp,
+                                 uint32 version) const
+{
+    auto const cp = lp.body.constantProduct();
+    if (!isAssetValid(cp.params.assetA))
+    {
+        return "LiquidityPool assetA is invalid";
+    }
+    if (!isAssetValid(cp.params.assetB))
```

Comment:
Missing one check after this one: `!(assetA < assetB)` is invalid

Reply:
Added